src/javascript/pages/cashier/top_up_virtual.js

```diff
@@ -19,7 +19,7 @@ const CashierTopUpVirtual = (function() {
         } else if (Client.is_logged_in() && !Client.is_virtual()) {
             viewError.removeClass('hidden')
                 .find('.notice-msg')
-                .text('Sorry, this feature is available to virtual accounts only.');
+                .text('This feature is not relevant to virtual-money accounts.');
```

Comment:
@negarn this is a check for logged in user and has real account `!Client.is_virtual()`